src/Platform/Proxy/ProxyBase.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <cassert>
+
```

Comment:
Docs here please.
ProxyBase used as wrapper to assert non null base class.

Reply:
Added docs.